quickstarts/spring-camel-drools-decision-table/pom.xml

```diff
@@ -240,12 +257,104 @@
       </plugin>
 
       <plugin>
+        <artifactId>maven-war-plugin</artifactId>
+        <executions>
+            <execution>
+                <id>default</id>
+                <phase>package</phase>
+                <goals>
+                    <goal>war</goal>
+                </goals>
+        <configuration>
```

Comment:
We should use consistent indents. Maybe it's just me, but I find the files more readable then.